ledger/controllers/surveyor.js

```diff
@@ -708,3 +739,15 @@ module.exports.initialize = async (debug, runtime) => {
     }
   }
 }
+
+function getVoteRate () {
+  return (runtime) => async (request, reply) => {
+    const surveyor = await server(request, reply, runtime)
+    console.log(surveyor)
```

Comment:
Do you intend to commit this log line?

Reply:
yeah, looks like i just forgot a step. was checking for where the altcurrency existed on the object. found it